### DISCUSSION TITLE
Line break in comment

### DISCUSSION THREAD

**Discussion by pixbalance** [Part-DB/Part-DB-server#763] [Q&A] [answered]
I am using partdb in a docker in unraid. So far it is working fine, but in the part description I cannot add line breaks.
If I copy formatted text it in the description it appears as plain text. Even then I cannot add line breaks afterwards.

**Comment by jbtronics** [comment, marked as answer]
That's intended. "Description" should be a short text describing the part in one line.

If you need to add multi-line texts, use the "Notes" field...
